docs-2.0/3.ngql-guide/15.text-based-index-statements/4.search-with-fulltext-index.md

```diff
@@ -0,0 +1,59 @@
+# Full-text search
+
+```ngql
+LOOKUP ON {<tag> | <edge_type>} WHERE <expression> [YIELD <return_list>]
+
+<expression>
+    PREFIX | WILDCARD | REGEXP |FUZZY
+
+<return_list>
+    <col_name> [AS <col_alias>] [, <col_name> [AS <col_alias>] ...]
+```
+
+Use the `LOOKUP ON` statement to do full-text search. The search string is specified in the `WHERE` clause.
+
+## Natural language full-text search
+
+A natural language search interprets the search string as a phrase in natural human language. The search is case-insensitive.
+
+## Examples
+
+```ngql
+nebula> LOOKUP ON t1 WHERE PREFIX(t1.c1, "a");
+nebula> LOOKUP ON t1 WHERE WILDCARD(t1.c1, "a");
+nebula> LOOKUP ON t1 WHERE REGEXP(t1.c1, "a");
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a");
+
+nebula> LOOKUP ON t1 WHERE PREFIX(t1.c1, "a", 1);
+nebula> LOOKUP ON t1 WHERE WILDCARD(t1.c1, "a", 1);
+nebula> LOOKUP ON t1 WHERE REGEXP(t1.c1, "a", 1);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 1);
+
+nebula> LOOKUP ON t1 WHERE PREFIX(t1.c1, "a", 1, 2);
+nebula> LOOKUP ON t1 WHERE WILDCARD(t1.c1, "a", 1, 2);
+nebula> LOOKUP ON t1 WHERE REGEXP(t1.c1, "a", 1, 2);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 1, 2);
+
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", AUTO, AND);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", AUTO, OR);
+
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 0, AND);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 0, OR);
+
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 0, OR, 1);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 0, OR, 1, 1);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 0, OR, -1, 1);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 0, OR, 1, -1);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 4, OR, 1, 1);
+
+nebula> LOOKUP ON t1 WHERE PREFIX(t1.c1, "a", AUTO, AND);
+```
+
+## Full-text search limitations
+
+For now, full-text search has the following limitations:
+
+- In elastic search, the maximum indexing string length is 256 bytes.
+- Multiple properties search is not supported in Nebula Graph 2.0.0-beta.
+- Ideographic language Chinese does not have word delimiters. Therefore, the built-in full-text parser cannot determine where words begin and end in these and other such languages. Install the [elasticsearch-analysis-ik](https://github.com/medcl/elasticsearch-analysis-ik) to parse Chinese.
```

Comment:
Ideographic languages such as written Chinese ...

where a phrase begins and ends in such a language?

Reply:
don't understand. but I think it's ok here.